demisto_sdk/commands/test_content/tests/test_context_test.py

```diff
@@ -129,3 +132,90 @@ def test_docker_thresholds_for_pwsh_integrations(mocker):
     memory_threshold, pid_threshold = test_context.get_threshold_values()
     assert memory_threshold == Docker.DEFAULT_PWSH_CONTAINER_MEMORY_USAGE
     assert pid_threshold == Docker.DEFAULT_PWSH_CONTAINER_PIDS_USAGE
+
+
+class TestPrintContextToLog:
+
+    @staticmethod
+    def create_playbook_instance(mocker):
+        test_playbook_configuration = TestConfiguration(generate_test_configuration(
+            playbook_id='playbook_with_context',
+            integrations=['integration']
+        ),
+            default_test_timeout=30)
+        pb_instance = TestPlaybook(mocker.MagicMock(), test_playbook_configuration)
+        pb_instance.build_context.logging_module = mocker.MagicMock()
+        return pb_instance
+
+    def test_print_context_to_log__success(self, mocker):
+        """
+        Given:
+            - A test context with a context_print_dt value
+            - The context result is empty
+        When:
+            - Running 'print_context_to_log' method
+        Then:
+            - Ensure that an empty result is being printed to the context
+        """
+        dt_result = "{'foo': 'goo'}"
+        expected_result = json.dumps(ast.literal_eval(dt_result), indent=4)
+        playbook_instance = self.create_playbook_instance(mocker)
+        client = mocker.MagicMock()
+        client.api_client.call_api.return_value = (dt_result, 200)
+        playbook_instance.print_context_to_log(client, incident_id='1')
+        assert playbook_instance.build_context.logging_module.info.call_args[0][0] == expected_result
+
+    def test_print_context_to_log__empty(self, mocker):
+        """
+        Given:
+            - A test context with a context_print_dt value
+            - The context result is empty
+        When:
+            - Running 'print_context_to_log' method
+        Then:
+            - Ensure that an empty result is being printed to the context
+        """
+        expected_dt = '{}'
+        playbook_instance = self.create_playbook_instance(mocker)
+        client = mocker.MagicMock()
+        client.api_client.call_api.return_value = (expected_dt, 200)
+        playbook_instance.print_context_to_log(client, incident_id='1')
+        assert playbook_instance.build_context.logging_module.info.call_args[0][0] == expected_dt
+
+    def test_print_context_to_log__none(self, mocker):
+        """
+        Given:
+            - A test context with a context_print_dt value
+            - The context result is None
+        When:
+            - Running 'print_context_to_log' method
+        Then:
+            - Ensure that an exception is raised and handled via logging error
+        """
+        expected_dt = None
+        expected_error = 'received error: malformed node or string: None for None'
```

Comment:
Sure, that's actually the error that comes from `ast`, but I can help make it clearer.

Reply:
👍🏼 So if you can I think it would be good to include info to help direct the person looking into it (while preserving the original error message in addition).